### PR TITLE
Misc. Changes & Fixes

### PR DESCRIPTION
**Pull request by N7Huntsman**
- Set mortar shells and cannon ammo to a new `HeavyAmmoBase` with a different tradeTag than small arms and add it as a separate category of trade item to small arms ammo. This allows the quantities to be better controlled, preventing massive explosions caused by traders carrying 500 90mm HE shells.
- Patch [HLX] UNSC Armory Heavy Laser Rifle / Spartan laser.
- Fix Laser Charge Pack description.
- Add `onehanded` tag to VWE flintlock pistol.
- Remove duplicate patch affecting the Royalty cape.

### CODE REVIEW

**Comment by Breadbox2k19** [comment]
5-20 is definitely too much for caravans, caravans would be overloaded can barely move. Consider 4-8 instead.

Please remember to add them to mod traders as well, if you go to the patch folder and search for 'trade', all of the trader related xml should surface.

**Comment by N7Huntsman** [comment]
> 5-20 is definitely too much for caravans, caravans would be overloaded can barely move.

As far as I'm aware, the trade caravans more or less ignore a lot of the weight/bulk restrictions that usually impact caravans--remember that they were previously carrying literally hundreds of these things when they spawned like smallarms ammo.

**Comment by Breadbox2k19** [comment]
> > 5-20 is definitely too much for caravans, caravans would be overloaded can barely move.
> 
> As far as I'm aware, the trade caravans more or less ignore a lot of the weight/bulk restrictions that usually impact caravans--remember that they were previously carrying literally hundreds of these things when they spawned like smallarms ammo.

They can still move, but the caravan animals move extremely slowly. It is best if they do not exceed their weight capacity.

File: Royalty/Patches/ThingDefs_Misc/Apparel_Royalty.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		<value>
```
Comment (Jalordon) [inline Royalty/Patches/ThingDefs_Misc/Apparel_Royalty.xml:18]: As long as you are touching Apparel_Royalty, could you also change the patch for Apparel_ShirtRuffle and Apparel_VestRoyal ; the combined patch only successfully modifies Apparel_RobeRoyal while the data structure for the other two can't accept the patch.

example of patching that did work when I used it:
```
	<Operation Class="PatchOperationReplace">
		<xpath>Defs/ThingDef[defName="Apparel_RobeRoyal"]/statBases/StuffEffectMultiplierArmor</xpath>
		<value>
			<StuffEffectMultiplierArmor>1</StuffEffectMultiplierArmor>
		</value>
	</Operation>
	
	<Operation Class="PatchOperationAdd">
		<xpath>Defs/ThingDef[defName="Apparel_ShirtRuffle"]</xpath>
		<value>
		<statBases>
			<StuffEffectMultiplierArmor>1</StuffEffectMultiplierArmor>
		</statBases>
		</value>
	</Operation>
	
	<Operation Class="PatchOperationAdd">
		<xpath>Defs/ThingDef[defName="Apparel_VestRoyal"]/statBases</xpath>
		<value>
			<StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
		</value>
	</Operation>
```
2 mult for vest to match corset,

**Comment by N7Huntsman** [comment]
> They can still move, but the caravan animals move extremely slowly. It is best if they do not exceed their weight capacity.

Trader caravan animals are already spawning with their weight capacity exceeded, I know know that this makes it meaningfully worse. Moreover, it appears that the lowest their speed will go is, for a muffalo, 2.50c/s (50% speed) when weighed down.

**Pull request merged**